.buildkite/test-pipeline.yaml

```diff
@@ -475,6 +475,18 @@ steps:
   - pytest -v -s distributed/test_pp_cudagraph.py
   - pytest -v -s distributed/test_pipeline_parallel.py
 
+- label: Disaggregated Prefill Test # 4min
```

Comment:
for a 4 minute test, please merge it into the existing 4 gpu test. it can save the ci cost.

Reply:
Oh ur right, let me do that.

Reply:
Done.